### PR TITLE
Some minor changes to the variant caller interpreter

### PR DESCRIPTION
**Pull request by susannasiebert**
- List the valid_callers explicitly in the plan files instead of having a default
- Add GatkSomaticIndel and Pindel to the list of valid_callers in the indel plan files

### CODE REVIEW

**Comment by tmooney** [comment]
+1

File: lib/perl/Genome/VariantReporting/plan_files/gold_germline_report_indels.yaml [hunk 1]
```diff
@@ -64,6 +64,10 @@ reports:
                 normal_sample_names:
                     - normal
             'variant-callers':
+                valid_callers:
+                    - VarscanSomatic
+                    - Sniper
+                    - Strelka
```
Comment (davidlmorton) [inline lib/perl/Genome/VariantReporting/plan_files/gold_germline_report_indels.yaml]: I think you missed the Indel callers here.

Comment (susannasiebert) [inline lib/perl/Genome/VariantReporting/plan_files/gold_germline_report_indels.yaml]: Fixed.

**Comment by davidlmorton** [comment]
+1

**Pull request merged**